.circleci/config.yml

```diff
@@ -61,6 +70,37 @@ jobs:
       - build-dockerfile:
           image-name: gas-oracle
           dockerfile: ./ops/docker/Dockerfile.gas-oracle
+  build-integration-tests:
+    docker:
+      - image: cimg/base:2021.04
+    steps:
+      - build-dockerfile:
+          image-name: integration-tests
+          target: integration-tests
+          dockerfile: ./ops/docker/Dockerfile.packages
+  deploy-nightly:
+    docker:
+      - image: cimg/base:2021.04
+    steps:
+      - gcp-gke/install
+      - gcp-gke/update-kubeconfig-with-credentials:
+          cluster: $STACKMAN_CLUSTER
```

Comment:
None of the others have a `$` prefix - is this intentional?

Reply:
Yes. `cluster` expects a string value, so we want the `$STACKMAN_CLUSTER` env var to be expanded prior to passing the argument to the `gcp-gke` orb. The other arguments expect an environment variable name, which will be expanded inside the orb.